src/Components/Fsi.fs

```diff
@@ -494,13 +496,27 @@ module Fsi =
             return terminal
         }
 
+    let private commentRegex =
+        Regex(@"\/{2,3}.+\S+", RegexOptions.ECMAScript)
+
+    let private multilineCommentRegex =
+        Regex(@"\(\*[\s\S]*?\*\)", RegexOptions.ECMAScript)
```

Comment:
Should these be compiled?

Reply:
@vzarytovskii these are compiled to JS regexes, and I don't think there's a 'compiled' equivalent on that runtime.

Reply:
Ah, I see